heareval/embeddings/runner.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+"""
+Computes embeddings on a set of tasks
+"""
+from pathlib import Path
+
+import click
+from tqdm import tqdm
+
+from heareval.embeddings.task_embeddings import Embedding, task_embeddings
+
+
+@click.command()
+@click.argument("module", type=str)
+@click.option(
+    "--model",
+    default=None,
+    help="Location of model weights file",
+    type=click.Path(exists=True),
+)
+@click.option(
+    "--tasks-dir",
+    default=None,
+    help="Location of tasks to compute embeddings on",
+    type=click.Path(exists=True),
+)
+def runner(module: str, model: str = None, tasks_dir: str = None) -> None:
+
+    # Check for directory containing the tasks
+    tasks_dir = Path("tasks") if tasks_dir is None else Path(tasks_dir)
```

Comment:
Why not just put that as the default in click?

Reply:
"tasks"

Reply:
I experimented with that but the error message if the tasks dir doesn't exist isn't very helpful so I did it this way. I think the error message here could be even more descriptive and the help on the tasks-dir command line arg should also say that this will by default looks for a directory called "tasks" in the cwd.